src/main/kotlin/com/tupocode/killerbot/configuration/TelegramBotConfiguration.kt

```diff
@@ -0,0 +1,25 @@
+package com.tupocode.killerbot.configuration
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Component
+
+@Component
+data class TelegramBotConfiguration(
+    @Value("{telegram.bot.token}")
+    var token: String? = null,
+
+    @Value("\${telegram.bot.base.url}")
+    var baseUrl: String? = null,
```

Comment:
remove all \$ escape characters. We don't need $ sighs here

Reply:
Okay, Sensei